tests/test_series.py

```diff
@@ -1236,3 +1236,7 @@ def cond2(x: pd.Series[int]) -> pd.Series[bool]:
 
     cond3 = pd.Series([False, True, True])
     check(assert_type(s.where(cond3, other=0), "pd.Series[int]"), pd.Series, int)
+    
+# from issue #348 (bitwise operators on Series should support int)
+s= pd.Series([1,2,3,4])
+check(assert_type(s & 3, pd.Series), pd.Series)
```

Comment:
A few comments here:
1. place the tests in a function, like the tests above it
2. add tests for the or operator ( `|` ) and xor operator (`^`)
3. add tests for the 3 reverse operators, e.g., `3 & s`

Reply:
Yes sir

Reply:
done with the changes sir